src/ui/Card.tsx

```diff
@@ -151,9 +236,15 @@ export function Card(props: CardProps): JSX.Element {
     subject: {
       fontWeight: 'bold',
       fontSize: 16,
-    }
+    },
   });
 
+  const cardStyleAnimated = {
+    card: {
```

Comment:
Nit: you could put this on cardStyle and call the property animatedCard or something. And maybe rename cardStyle to style.